js/modules/Debug/Debug.js

```diff
@@ -536,13 +536,22 @@ window.GLPI.Debug = new class Debug {
                 (filtered_request_id !== undefined && filtered_request_id !== request_id)) {
                 return;
             }
+            const clipboard_title = __("Copy query to clipboard");
```

Comment:
Since this is debug, this should not be translated (nothing else seems translated in this file).
```suggestion
            const clipboard_title = "Copy query to clipboard";
```